### PR TITLE
Create MM-CCK-Lights.cfg

### PR DESCRIPTION
**Pull request by zer0Kerbal**
if CommunityCategoryKit is installed, move parts into Light Category instead of Utility. Could also just merge into parts by adding cck-lights to tags

### CODE REVIEW

File: Gamedata/TrackingLights/MM_Patches/MM-CCK-Lights.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@PART[MiniSpotlight,LaunchPadLamp]:NEEDS[TrackingLights,CommunityCategoryKit]
+{
+	@tags ^= :^:cck-lights
```
Comment (zer0Kerbal) [inline Gamedata/TrackingLights/MM_Patches/MM-CCK-Lights.cfg:3]:
```suggestion
	tags = cck-lights light shine spot torch track
```

Comment (zer0Kerbal) [reply Gamedata/TrackingLights/MM_Patches/MM-CCK-Lights.cfg:3]: reason - didn't notice part.cfg didn't already have a tag section

Comment (zer0Kerbal) [reply Gamedata/TrackingLights/MM_Patches/MM-CCK-Lights.cfg:3]: could also just add "tags = cck-lights light shine spot torch track" to each part.cfg - suggest just after "description"

**Comment by droric** [comment]
Looks good.  Thanks for the PR

**Pull request merged**